pubspec.yaml

```diff
@@ -93,59 +93,59 @@ dependencies:
   # Add hex color codes to your flutter projects
   hexcolor: ^3.0.1
   # A composable, multi-platform, Future-based API for HTTP requests.
-  http: ^0.13.0
+  http: ^1.1.0
   # Open source icon sets for flutter
   iconify_flutter: ^0.0.5
   # Convenient code generator for get_it
   injectable: ^2.3.0
   # Allow you to have a screen at launcher for example
-  introduction_screen: ^3.1.8
+  introduction_screen: ^3.1.7
   # Extremely fast, easy to use, and fully async NoSQL database for Flutter.
-  isar: ^3.1.0+1
+  isar: ^3.1.0
   # Isar Core binaries for the Isar Database. Needs to be included for Flutter apps.
-  isar_flutter_libs: ^3.1.0+1
+  isar_flutter_libs: ^3.1.0
   # Defines the annotations used by json_serializable to create code for JSON serialization and deserialization.
-  json_annotation: ^4.8.1
+  json_annotation: ^4.8.0
   # Dart Build System builders for handling JSON.
-  json_serializable: ^6.7.0
+  json_serializable: ^6.6.1
   # Port of Kotlin's Kotlin Standard library for Dart/Flutter projects
-  kt_dart: ^1.1.0
+  kt_dart: ^1.0.0
   # Liquid progress indicator for Flutter.
-  liquid_progress_indicator_v2: ^0.5.0
+  liquid_progress_indicator: ^0.4.0
```

Comment:
This package is unsupported and didn't got update in 2 years 
![image](https://github.com/CyBear-Jinni/cbj_app/assets/9304740/77e37da0-1ff5-4a1b-b6bf-bab4bdb3e2bf)

Please keep v2 version

Reply:
Also I see that you started working on this branch and forked master instead of dev.
Please fork dev so that you will not add unwanted code that has already been solved in the flutter upgrade process.

Reply:
Ok will do the necessary changes immediately